src/v/cluster/rm_stm.h

```diff
@@ -390,6 +390,8 @@ class rm_stm final : public persisted_stm {
         // by spec should be ready for thier commands being rejected so it's
         // ok by design to have false rejects
         absl::flat_hash_map<model::producer_identity, seq_entry> seq_table;
+
+        absl::flat_hash_map<model::producer_identity, model::tx_seq> tx_seqs;
```

Comment:
since this is another piece of in memory state added to the `log_state` do we have any accounting on how much memory will the state need ? Since it is created for each partition and it contains 8 containers it may be a pretty large amount

Reply:
no not really, we do expire old producer_identity but it happens with large period by default; I created an issue to follow up it later: https://github.com/redpanda-data/redpanda/issues/6921